dist/runCall.js

```diff
@@ -89,19 +89,30 @@ module.exports = function (opts, cb) {
         case GETACCOUNT:
           {
             console.log('GETACCOUNT');
-            computeGetAccount(query);
+            computeGetAccount(query, function (fromAccount) {
+              var account = createAccount(fromAccount);
+              client.write(account);
+            });
             break;
           }
         case GETSTORAGEDATA:
           {
             console.log('GETSTORAGEDATA');
-            computeGetStorageData(query);
+            computeGetStorageData(query, function (value) {
+              var storageData = createStorageData(value.current);
+              client.write(storageData);
+            });
             break;
           }
         case GETCODE:
           {
             console.log('GETCODE');
-            computeGetCode(query);
+            computeGetCode(query, function (contractCode, compiled) {
+              isCompiled = compiled;
```

Comment:
Currently, `isCompiled` is not used anywhere. We will have to decide if/how use it.